examples/gazebo_tank/run.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-trap 'kill %1' SIGINT
+trap 'kill %1' INT
```

Comment:
What's the difference here? I don't think I've ever actually used ``trap`` in bash but googled examples seem to use ``SIGINT``

Reply:
Apparently they're equivalent, but the standard POSIX way of doing it is to use ``INT`` instead. Nitik is using zsh which doesn't like ``SIGINT``